ex1/ex1/Exercises/Exercise1OOP.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ex1.Exercises
+{
+    public class Student
+    {
+        public string _name;
+        public string _surname;
+        public string _dateOfBirth;
+        private string _grade;
+        private double _mark;
+
+        public Student(string name, string surname, string dateOfBirth)
```

Comment:
Constructor should initialize values for all private fields. Please add also **grade** and **marks** to this special constructor